newdle/client/src/components/creation/timeslots/Timeline.js

```diff
@@ -370,9 +396,13 @@ export default function Timeline({date, availability, defaultMinHour, defaultMax
                 <div className={styles['config-box']}>
                   <DurationPicker />
                   <TimezonePicker
-                    onChange={value => dispatch(setTimezone(value))}
+                    onChange={value => {
+                      dispatch(setTimezone(value));
+
```

Comment:
so much blank space here! 👾

Reply:
Haha :D I thought it would be better to separate these calls but I don't mind changing it :D